test/paper-dialog-scrollable.html

```diff
@@ -219,6 +235,36 @@
           });
         });
 
+        test('shows bottom divider if content is added dynamically', function(done) {
+          var container = fixture('dynamic-content');
```

Comment:
Fair enough, I was looking to recreate the scenario but it can be made simpler indeed.